nixos/modules/services/web-apps/nextcloud.nix

```
@@ -64,6 +66,10 @@ in {
       default = false;
       description = "Use https for generated links.";
     };
+    package = mkOption {
+      type = types.package;
+      description = "Which package to use for the Nextcloud instance.";
+    };
```

```suggestion
    relatedPackages = [ "nextcloud17" "nextcloud18" ];
    };
```
RHS can probably be extracted from `pkgs`.